.github/workflows/haskell.yml

```diff
@@ -8,13 +8,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc: [ '8.0.2', '8.2.2', '8.4.4', '8.6.5', '8.8.4', '8.10.7', '9.0.1', '9.2.1' ]
+        ghc: [ '8.0.2', '8.2.2', '8.4.4', '8.6.5', '8.8.4', '8.10.7', '9.0.2', '9.2.4', '9.4.2' ]
     steps:
     - uses: actions/checkout@v2
     - uses: haskell/actions/setup@v1
       with:
         ghc-version: ${{ matrix.ghc }}
-        cabal-version: '3.4'
+        cabal-version: '3.8'
```

Comment:
It doesn't seem to be available yet

Reply:
@fumieval Oh I should've updated gh action version as well. Now it should be okay, I tested it in my fork https://github.com/marshall-lee/word-compat/pull/1

Reply:
Great, LGTM